extensions/1loc/package.json

```
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "1loc",
+  "title": "1loc",
+  "description": "Browsing 1loc.dev",
+  "icon": "command-icon.png",
+  "author": "kud",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "1loc",
```

```suggestion
      "title": "Search Javascript Utilities",
      "subtitle": "1LOC",
```
I'm not sure about the title, but something like that should be changed 😄